<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QMainWindow" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings:</string>
    </property>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPackCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cigarretes in a pack :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPackCount">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPackCost">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblPackCost">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cost of a pack :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editTimerPeriod">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblTimer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timer Period to alert ( min ) :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMessage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Popup Message Display:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSound">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sound Play:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editGramOnce">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblGramCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cannabis at a time(gram) :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCostGram">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblGramCost">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cannabis Cost per gram :</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupSound">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioSoundN">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;No</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioSoundY">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Yes</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupMessage">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioMessageN">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioMessageY">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Yes</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="editBackgroudImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelectImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Backgoud Image</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
